gradio/interface.py

```diff
@@ -477,80 +556,7 @@ def __repr__(self):
         return repr
 
     def get_config_file(self):
```

Comment:
What about returning the `self.blocks` in the `__init__` so that `Interface` is a subclass of `Blocks`? In that case, you wouldn't even have to define these methods, you would just inherit them from `Blocks`

Reply:
The return type of init is ignored so just returning the blocks won’t work. subclassing could work, I’ll try

Reply:
Done.